JSDemos/Demos/List/GroupedList/description.md

```diff
@@ -1 +1,11 @@
-The **List** UI component can display items in&nbsp;groups if&nbsp;they are arranged so&nbsp;in&nbsp;the data source. Groups have captions, which are customized in&nbsp;this demo.
\ No newline at end of file
+It is possible to display the **List** in groups under the following conditions:
```

Comment:
"display the list in groups" is a vague statement. Data Grid can display data in groups - and there can be unlimited number of hierarchy levels. I think we need to be specific here that we're strictly speaking about two-level hierarchies. It's a simple parent-child relationship. 

I also think it's best to restructure the information a little. We can mention the property right away and then we can clearly separate two ways to supply grouped data (using sub-headers). 

```suggestion
**List** can display items as a two-level hierarchy - parent and child items. To group items, set the [grouped](/Documentation/ApiReference/UI_Components/dxList/Configuration/#grouped) property to **true** and make sure that the data source defines the hierarchy in one of the following ways. 
```